src/constants.ts

```diff
@@ -17,3 +17,7 @@ export const CHILD_REGISTER_ENCOUNTER_TYPE = "";
 //maternity
 export const ADMISSION_DATE = "cb9b223e-a09c-4f8a-b34d-a0934a2af04a";
 export const DELIVERY_TYPE = "dcc3ac63-30ab-102d-86b0-7a5022ba4115";
+
+// opd
+export const OUTPATIENT_REGISTER_ENCOUNTER_TYPE =
```

Comment:
@Irenyak1 please rename this OUTPATIENT_DEPARTMENT_ENCOUNTER_TYPE. In a facility setting, there is not encounter type known as Outpatient register encounter type

Reply:
@solemabrothers , ok let me do that